### DISCUSSION TITLE
an error in using git hub education

### DISCUSSION THREAD

**Discussion by farokyosry** [community/community#111377] [GitHub Education] [answered]
### Hi everyone,

I tried more than once to enter the university’s photo, but it would take me back to the page again and not write anything to me.

**Comment by ettaboyle** [comment]
Hi @farokyosry 👋🏻 

The team just updated the [Student Developer Pack Application FAQs & Common Rejection Reasons (Updated!)](https://github.com/orgs/community/discussions/111352). Please check out that post for guidance and suggestions on all things related to the Student Developer Pack. 

We'll share your feedback on your experience (described above) with the Education team in case this is a product related issue.

Thank you!

**Reply by ExiledShurtugal** [reply, marked as answer]
Hey @farokyosry ,

I was experiencing the exact same issue. Finally got it to work by using my pc camera to take the photo instead of using a photo from my phone. Try **different devices and different photos**.
